components/dashboard/src/whatsnew/WhatsNew-2021-06.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { User } from "@gitpod/gitpod-protocol";
+import { WhatsNewEntry } from "./WhatsNew";
+import { switchToVSCodeAction } from "./WhatsNew-2021-04";
+import CodeText from "../components/CodeText";
+import PillLabel from "../components/PillLabel";
+
+export const WhatsNewEntry202106: WhatsNewEntry = {
+    children: (user: User, setUser: React.Dispatch<User>) => {
+
+        return <>
+            <div className="border-t border-b border-gray-200 dark:border-gray-800 -mx-6 px-6 pt-6 pb-4">
+                <p className="pb-2 text-gray-900 dark:text-gray-100 text-base font-medium">Exposing Ports <PillLabel>Configuration Update</PillLabel></p>
+                <p className="pb-2 text-gray-500 dark:text-gray-400 text-sm">We've changed the default behavior of exposed ports to improve the security of your dev environments.</p>
```

Comment:
**question**: Would you consider the following change to be more accurate? Your call! 

```suggestion
                <p className="pb-2 text-gray-500 dark:text-gray-400 text-sm">We've changed the default behavior of exposed ports to improve the security of your workspaces.</p>
```

Reply:
Both sound good. I'm for keeping “dev environments”. Like a tad more.

Reply:
DEAL